### PR TITLE
Update to Electron from 17.1.0 18.0.3

### PR DESCRIPTION
**Pull request by raksooo**
This PR:
* Updates Electron from version 17.1.0 to 18.0.3
* Updates NodeJS to 16.13.2 to match Electron
* Makes GitHub Action node setup use the value in `.nvmrc`

Electron 18 changelog:
https://github.com/electron/electron/releases/tag/v18.0.0

Git checklist:

* [x] Describe the change in **`CHANGELOG.md`** under the `[Unreleased]` header.
* [x] Check that commits follow the [Mullvad coding guidelines](https://github.com/mullvad/coding-guidelines)

---
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501)

### CODE REVIEW

**Review by pronebird — CHANGES_REQUESTED** [review]
> Reviewed 2 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501)* status: 2 of 6 files reviewed, 1 unresolved discussion (waiting on @raksooo)

___
*[.github/workflows/translations.yml, line 20 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501#-N-XuqSg4FfT4BbGAgaT:-N-XuqSg4FfT4BbGAgaU:bazbwtw) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/dc5e1b03616e1ee79cb546a46c774982fb9a63a4/.github/workflows/translations.yml#L20)):*
> ```YAML
> 
>             - name: Read .nvmrc
>               run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
> ```

Don't you have to use double quotes for the echo message?, i.e:

```
echo "::set-output name=NODE_VERSION::$(cat .nvmrc)"
```

**Review by pronebird — COMMENTED** [review]
> Reviewed 4 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501)* status: all files reviewed, 1 unresolved discussion (waiting on @raksooo)

**Review by raksooo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501)* status: all files reviewed, 1 unresolved discussion (waiting on @pronebird)

___
*[.github/workflows/translations.yml, line 20 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501#-N-XuqSg4FfT4BbGAgaT:-N-Y8yRK8Vng7geGpS1Z:b-qtlom6) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/dc5e1b03616e1ee79cb546a46c774982fb9a63a4/.github/workflows/translations.yml#L20)):*
<details><summary><i>Previously, pronebird (Andrej Mihajlov) wrote…</i></summary><blockquote>

Don't you have to use double quotes for the echo message?, i.e:

```
echo "::set-output name=NODE_VERSION::$(cat .nvmrc)"
```
</blockquote></details>

I copied it from some guide about this and it seems to work. Here's the job in GitHub actions

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -22,10 +22,14 @@ jobs:
...
-            - name: Setup Node.js environment
+            - name: Read .nvmrc
+              run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+              id: nvm
+
+            - name: Setup Node.js ${{ steps.nvm.outputs.NODE_VERSION }}
               uses: actions/setup-node@v3
               with:
-                  node-version: '16.9.1'
+                  node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
```
Comment (MrChocolatine) [inline .github/workflows/frontend.yml:32]: You do not need a separate step to read the file `.nvmrc`:
https://github.com/actions/setup-node/blob/v3.1.1/docs/advanced-usage.md#node-version-file

```suggestion
                  node-version-file: '.nvmrc'
```

Comment (raksooo) [reply .github/workflows/frontend.yml]: Thanks! I didn't know about that option :+1:

**Review by raksooo — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501)* status: 1 of 6 files reviewed, 3 unresolved discussions (waiting on @MrChocolatine, @pronebird, and @raksooo)

___
*[.github/workflows/translations.yml, line 20 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501#-N-XuqSg4FfT4BbGAgaT:-N-b7VTb95gvLwLNaPD-:b-xb2oxc) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/dc5e1b03616e1ee79cb546a46c774982fb9a63a4/.github/workflows/translations.yml#L20)):*
<details><summary><i>Previously, raksooo (Oskar Nyberg) wrote…</i></summary><blockquote>

I copied it from some guide about this and it seems to work. Here's the job in GitHub actions 


</blockquote></details>

This code has been removed now.

**Review by MrChocolatine — APPROVED** [review]

**Review by pronebird — APPROVED** [review]
:lgtm:

> Reviewed 4 of 5 files at r3.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501#-:-N-bGIZt5MpcPcq3Au4v:bnfp4nl)* status: 5 of 6 files reviewed, 2 unresolved discussions (waiting on @MrChocolatine, @pronebird, and @raksooo)

___
*[.github/workflows/frontend.yml, line 32 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501#-N-Xt-gm5QCm5Chysl7A-r1-32:-N-bG0rcEvzbXd2TF5rs:b-my9gk) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/dc5e1b03616e1ee79cb546a46c774982fb9a63a4/.github/workflows/frontend.yml#L32)):*
<details><summary><i>Previously, raksooo (Oskar Nyberg) wrote…</i></summary><blockquote>

Thanks! I didn't know about that option :+1:
</blockquote></details>

That's neat.

**Review by pronebird — COMMENTED** [review]
> Dismissed @MrChocolatine from a discussion.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/3501)* status: 5 of 6 files reviewed, 1 unresolved discussion (waiting on @MrChocolatine and @pronebird)

**Pull request merged**